### PR TITLE
Change url manifest

### PR DESCRIPTION
**Pull request by DarioLodeiros**

### CODE REVIEW

**Comment by FerNoailles** [comment]
I don't agree

File: helpdesk_curso/__manifest__.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
     'license': "AGPL-3",
     'maintainer': 'Fernando Vasconcellos',
     'category': 'Helpdesk',
-    'website': 'https://odoo-community.org/',
```
Comment (DarioLodeiros) [inline helpdesk_curso/__manifest__.py:9]: Coment test

File: helpdesk_curso/__manifest__.py [hunk 2]
```diff
@@ -16,4 +16,4 @@
         'security/helpdesk_security.xml',
         ],
     'installable': True,
```
Comment (DarioLodeiros) [inline helpdesk_curso/__manifest__.py:18]: Comment 2

**Review by DarioLodeiros — COMMENTED** [review]
Can you please review this?

File: helpdesk_curso/__manifest__.py (hunk 2 above)
Comment (DarioLodeiros) [reply helpdesk_curso/__manifest__.py:18]: done

File: helpdesk_curso/__manifest__.py (hunk 1 above)
Comment (DarioLodeiros) [reply helpdesk_curso/__manifest__.py:9]: done

**Pull request closed without merging**